### PR TITLE
Port commands to new command manager

### PR DESCRIPTION
**Pull request by Sepshun**

### CODE REVIEW

File: src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package xyz.atrius.waystones.commands
+
+import org.bukkit.Bukkit
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import xyz.atrius.waystones.utility.defaultWarpKey
+import xyz.atrius.waystones.utility.pluralize
+import xyz.atrius.waystones.utility.translateColors
+
+object GetKeyCommand : SimpleCommand("getkey", "gk", "key") {
```
Comment (AtriusX) [inline src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt:10]: lets add `k` as a command alias for this

File: src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        val (player, amount) = getPlayerAndAmount(
+            sender as Player,
+            args.getOrNull(0) ?: "",
+            args.getOrNull(1) ?: ""
+        )
+        // Check Permissions
+        if (
+            !sender.hasPermission("waystones.getkey.self") || // Give Key to Self
+            (!sender.hasPermission("waystones.getkey.all") && sender != player)
+        ) { // Give Key to Other
```
Comment (AtriusX) [inline src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt:29]: Perhaps we should format it like this: 
```kt
		// Give Key to other players
        if (!sender.hasPermission("waystones.getkey.self") 
        	|| !sender.hasPermission("waystones.getkey.all") 
	        && sender != player
        ) {
```

File: src/main/kotlin/xyz/atrius/waystones/commands/InfoCommand.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package xyz.atrius.waystones.commands
+
+import org.bukkit.command.CommandSender
+import xyz.atrius.waystones.plugin
+import xyz.atrius.waystones.utility.translateColors
+
+object InfoCommand : SimpleCommand("info") {
```
Comment (AtriusX) [inline src/main/kotlin/xyz/atrius/waystones/commands/InfoCommand.kt:7]: add `i` as an alias for this

File: src/main/kotlin/xyz/atrius/waystones/Waystones.kt [hunk 4]
```diff
@@ -41,6 +43,8 @@ class Waystones : KotlinPlugin() {
 //        )
         registerNamespaces(
             CommandNamespace("waystones").register(
```
Comment (AtriusX) [inline src/main/kotlin/xyz/atrius/waystones/Waystones.kt:45]: lets add `ws` as an alias for the namespace

**Review by AtriusX — CHANGES_REQUESTED** [review]
It looks good overall, though there are a couple small areas that could be touched up a bit.

File: src/main/kotlin/xyz/atrius/waystones/Waystones.kt (hunk 4 above)
Comment (Sepshun) [reply src/main/kotlin/xyz/atrius/waystones/Waystones.kt:45]: this is already being handled by bukkit/spigot for us within the plugin.yml and i've just confirmed it does work

Comment (AtriusX) [reply src/main/kotlin/xyz/atrius/waystones/Waystones.kt:45]: I see, I guess that is fine for now then.

File: src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt (hunk 2 above)
Comment (Sepshun) [reply src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt:29]: i feel this wouldn't function as intended. two criteria are being checked here and if i'm not mistaken the extra pair of parenthesis are important to that check

i won't deny the formatting is difficult to read as is, but this seems equally as hard and function in unintended ways

Comment (AtriusX) [reply src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt:29]: The extra set of parentheses was deemed unnecessary by intelliJ. Given that the operations are checked sequentially and the order given this sorta makes sense as to why.

Comment (Sepshun) [reply src/main/kotlin/xyz/atrius/waystones/commands/GetKeyCommand.kt]: seems i was mistaken about how kotlin handles this, this does indeed work as intended, i've pushed the change

**Review by AtriusX — APPROVED** [review]
Everything looks good!

**Pull request merged**